docs/static/demo/android/allure-results/environment.xml

```diff
@@ -1 +1 @@
filtering</key><value>FilteringConfiguration(allowlist=[], blocklist=[])</value></parameter><parameter><key>ignoreFailures</key><value>false</value></parameter><parameter><key>isCodeCoverageEnabled</key><value>true</value></parameter><parameter><key>executionStrategy</key><value>ExecutionStrategyConfiguration(mode=ANY_SUCCESS, fast=true)</value></parameter><parameter><key>includeSerialRegexes</key><value>[]</value></parameter><parameter><key>excludeSerialRegexes</key><value>[]</value></parameter><parameter><key>testBatchTimeoutMillis</key><value>1800000</value></parameter><parameter><key>testOutputTimeoutMillis</key><value>300000</value></parameter><parameter><key>debug</key><value>true</value></parameter><parameter><key>screenRecordingPolicy</key><value>ON_FAILURE</value></parameter><parameter><key>vendorConfiguration</key><value>AndroidConfiguration(androidSdk=/home/pkunzip/.cache/android-sdk, applicationOutput=/home/pkunzip/Development/marathon/sample/android-app/app/build/outputs/apk/debug/app-debug.apk, testApplicationOutput=/home/pkunzip/Development/marathon/sample/android-app/app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk, splitApks=null, extraApplicationsOutput=null, outputs=null, autoGrantPermission=true, instrumentationArgs={}, applicationPmClear=false, testApplicationPmClear=false, adbInitTimeoutMillis=30000, installOptions=, serialStrategy=AUTOMATIC, screenRecordConfiguration=ScreenRecordConfiguration(preferableRecorderType=null, videoConfiguration=VideoConfiguration(enabled=true, width=720, height=1280, bitrateMbps=1, timeLimit=180, timeLimitUnits=SECONDS), screenshotConfiguration=ScreenshotConfiguration(enabled=true, width=720, height=1280, delayMs=500)), waitForDevicesTimeoutMillis=30000, allureConfiguration=AllureConfiguration(enabled=true, relativeResultsDirectory=/files/allure-results, pathRoot=APP_DATA), timeoutConfiguration=TimeoutConfiguration(shell=PT20S, listFiles=PT20S, pushFile=PT1M, pushFolder=PT1M, pullFile=PT30S, uninstall=PT20S, install=PT1M, screenrecorder=PT3M20S, screencapturer=PT0.3S, socketIdleTimeout=PT30S, portForward=PT20S, boot=PT30S), fileSyncConfiguration=FileSyncConfiguration(pull=[FileSyncEntry(relativePath=files/allure-results, pathRoot=APP_DATA, aggregationMode=TEST_RUN), FileSyncEntry(relativePath=/files/allure-results, pathRoot=APP_DATA, aggregationMode=TEST_RUN), FileSyncEntry(relativePath=coverage, pathRoot=APP_DATA, aggregationMode=POOL)], push=[]), threadingConfiguration=ThreadingConfiguration(bootWaitingThreads=4, adbIoThreads=4), testParserConfiguration=RemoteTestParserConfiguration(instrumentationArgs={listener=com.malinskiy.adam.junit4.android.listener.TestAnnotationProducer}), testAccessConfiguration=TestAccessConfiguration(adb=true, grpc=true, console=true, consoleToken=), adbServers=[AdbEndpoint(host=127.0.0.1, port=5037)], disableWindowAnimation=true)</value></parameter><parameter><key>deviceInitializationTimeoutMillis</key><value>180000</value></parameter></environment>
```

Comment:
As mentioned before, this should not be included since there are no changes to the actual allure data files. The only case where this should change is when there is a new allure report demo which is not the case here

Reply:
@Malinskiy I already revert all changes to master , you can check by compare two branches for this file, but I don't know the reason why we see it here , and we have a lot of merges of master to this branch and I can't just revert all to recreate the commit without this file